documentation/specs/proposed/BuildCheck.md

```diff
@@ -128,9 +130,20 @@ We might as well consider specifying custom analyzers on a command line (as a no
 
 There will be 3 mechanisms of configuring the analyzers and rules:
 * The default configuration declared by the analyzers themselves ([more details on implementation](#rules-declaration))
-* [Sdk Analysis Level property](https://github.com/dotnet/designs/blob/main/proposed/sdk-analysis-level.md) – mostly for the inbox analyzers
+* The TFM of the project and the [Sdk Analysis Level property](https://github.com/dotnet/designs/blob/main/proposed/sdk-analysis-level.md) – mostly for the inbox analyzers
 * `.editorconfig` file
 
+We will also consider respecting `SdkAnalysisLevel` to override the per-TFM defaults. Additionally we may use the existing `RunAnalyzers` "master switch" property to make it possible to disable everything whole-sale. This would be used in scenarios like F5 in VS.
```

Comment:
Do you have any suggestion on how the TFM and SdkAnalysisLevel would be combined? E.g. SdkAnalysisLevel is respected only from specific TFM up?

Not importnat now... but if you already have some vision - it might be nice to capture it

Reply:
If we use `SdkAnalysisLevel` I think it would be good to align the behavior with the rest of the SDK. I'm hesitant to put a stake in the ground now when `SdkAnalysisLevel` is effectively unused. So - I would prefer to leave this open for now and have a wider discussion later, most likely early next year.

Reply:
Roslyn is going to start using it this summer with their 'backing field' breaking change, so we can look at that and evaluate behaviors/decisions based on how their work plays out.